talosintelligence.json

```
@@ -208,7 +218,7 @@
             "action": "url reputation",
             "identifier": "url_reputation",
             "description": "Queries URL info",
-            "verbose": "",
+            "verbose": "Provides information on an url's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```

Do for all the other `verbose` where it is Provides-->Provide
```suggestion
            "verbose": "Provide information on an url's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```